db/migrate/20220827142531_add_fixed_to_event_streams.rb

```diff
@@ -0,0 +1,5 @@
+class AddFixedToEventStreams < ActiveRecord::Migration[6.0]
+  def change
+    add_column :event_streams, :fixed, :string
```

Comment:
@OrGur1987 what does `fixed` mean in the context of an event?

Reply:
it means that the problem which caused the event has been resolved

Reply:
Hm that sounds more like a different event type to me, I'm heavily leaning against this change since it isn't relevant for all of the other events that we have

Reply:
NOTE we have a "chain_id" if you wanted to associate a "failed" event with a later "fixed" event, you can have the event type like "storage_drive_failure" and then "storage_drive_fixed"